rpm/389-ds-base.spec.in

```diff
@@ -386,6 +386,7 @@ Requires: python%{python3_pkgversion}-dateutil
 Requires: python%{python3_pkgversion}-argcomplete
 Requires: python%{python3_pkgversion}-libselinux
 Requires: python%{python3_pkgversion}-setuptools
+Requires: python%{python3_pkgversion}-cryptography
```

Comment:
I think, we also need to add the dependency here:
https://github.com/389ds/389-ds-base/blob/main/src/lib389/requirements.txt

And here:
https://github.com/389ds/389-ds-base/blob/main/src/lib389/setup.py#L94

Reply:
Well it's not an install requirement, but a runtime requirement.  I'm adding it anyway, but I'm not sure we actually need it in setup.py

Reply:
Fixed